dashboard/lib/public/javascripts/dashboard.js

```diff
@@ -164,22 +164,21 @@ function loadReport(filename) {
 }
 
 
+function asyncLogout() {
+    setTimeout(function() {
+        Fiware.signOut('health')
```

Comment:
Referred to @pratid comment: I can't to write a config file, because the code will run in navigator.